appeals/web/src/server/appeals/appeal-details/service-user/service-user.controller.js

```diff
@@ -0,0 +1,92 @@
+import { getAppealDetailsFromId } from '#appeals/appeal-details/appeal-details.service.js';
+import logger from '#lib/logger.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+import { changeServiceUserPage } from './service-user.mapper.js';
+import { updateServiceUser } from './service-user.service.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeServiceUser = async (request, response) => {
+	return renderChangeServiceUser(request, response);
+};
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderChangeServiceUser = async (request, response) => {
+	const {
+		errors,
+		params: { appealId, userType }
+	} = request;
+
+	const appealDetails = await getAppealDetailsFromId(request.apiClient, appealId);
+	if (userType in appealDetails) {
```

Comment:
Probably worth adding a check to render a 500 page if the appealDetails wasn't found, eg. `if (appealDetails && userType in appealDetails) { ...`